src/app/components/PressCardList/Item/PressCardItem.tsx

```diff
@@ -0,0 +1,45 @@
+'use client'
+
+import Image from 'next/image'
+import { useFormattedDate } from '../../../../helpers/hooks/useFormattedDate'
+import { ArrowLink } from '@/components/ui/ArrowLink/ArrowLink'
+
+import { IPressCard } from '@/helpers/types/pressItem'
+import styles from './PressCardItem.module.css'
+
+export const PressCardItem = ({
+  title,
+  outline,
+  publisher,
+  publishDate,
+  publishUrl,
+  imageUrl,
+}: IPressCard) => {
+  const { formattedDate } = useFormattedDate(publishDate, 'PressCard')
+
+  return (
+    <li className={styles.root}>
+      <div className={styles.header}>
+        <span className={styles.date}>{formattedDate}</span>
+        <h3 className={styles.title}>{title}</h3>
+      </div>
+      <div className={styles.body}>
+        <p className={styles.outline}>{outline.slice(0, 200) + '...'}</p>
+        <ArrowLink
+          href={publishUrl}
+          text={`Read on ${publisher}`}
+          className={styles.link}
+        />
+
+        <div className={styles.imgWrapper}>
+          <Image
+            fill={true}

```

Comment:
@Yura33-dev For boolean props, we only need to specify the prop name without passing `true` as its value. The presence of the prop automatically means true, and if the boolean prop is not specified, it defaults to false.

```suggestion
            fill
```

Additionally, when using the Next.js Image component with the `fill` attribute, it's recommended to provide information about how wide the image will be at different breakpoints via the `sizes` prop, which can greatly improve performance. You can read more about it in the [Next.js documentation](https://nextjs.org/docs/app/api-reference/components/image#sizes)